File: SafetyInspectionApplication/app/src/main/java/com/kuanquan/safetyinspectionapplication/TestApplication.java
package com.kuanquan.safetyinspectionapplication;

import android.app.Application;

import com.lahm.library.EasyProtectorLib;

public class TestApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        EasyProtectorLib.checkXposedExistAndDisableIt();
//        可以在启动时创建localServerSocket
//        EasyProtectorLib.checkIsRunningInVirtualApk(getPackageName(), new VirtualCheckCallback() {
//            @Override
//            public void findSuspect() {
//                System.exit(0);
//            }
//        });
    }
}

File: Animator/sinaweibopraise/src/main/java/com/kuanquan/sinaweibopraise/TestApplication.kt
package com.kuanquan.sinaweibopraise

import android.app.Application
import android.content.Context

class TestApplication: Application() {

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
    }

    override fun onCreate() {
        super.onCreate()
        AppLifecycleManager.init(this)
    }
}